synapse/tests/files/stormcov/lookup.storm

```diff
@@ -3,3 +3,4 @@ function foo() {
     stop
 }
 yield $foo()
+yield $foo()
```

Comment:
What's this dood about?

Reply:
this is coverage for the `if info is None:` check that was added

Reply:
it needed to call it a second time to hit that check